iseq.c

```diff
@@ -325,9 +325,13 @@ rb_iseq_mark(const rb_iseq_t *iseq)
         if (body->call_data) {
```

Comment:
from commit log:
> "to increase GC pressure"

to reduce?

Reply:
ah, does it means "on the fly CI creation will increase the pressure"?

Reply:
Yes.
![Screenshot from 2020-06-03 18-04-28](https://user-images.githubusercontent.com/15377/83617918-bb229400-a5c4-11ea-9148-d03d108be40b.png)

Reply:
The purpose is not increasing GC pressure...

Reply:
Sorry about that.  Let me rephrase in a moderate tone (will amend the commit message).